easybuild/easyconfigs/s/SQLite/SQLite-3.8.8.1-intel-2015a.eb

```diff
@@ -0,0 +1,40 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'SQLite'
+version = '3.8.8.1'
+
+homepage = 'http://www.sqlite.org/'
+description = 'SQLite: SQL Database Engine in a C Library'
+
+# eg. http://www.sqlite.org/2014/sqlite-autoconf-3080600.tar.gz
+source_urls = ['http://www.sqlite.org/2015/']
+version_numbers = version.split('.')
+main_version = version_numbers.pop(0)
+version_str = '%s%s' % (main_version, ''.join('%02d' % int(x) for x in version_numbers))
+sources = ['sqlite-autoconf-%s.tar.gz' % version_str]
+
+toolchain = {'name': 'intel', 'version': '2015a'}
```

Comment:
minor style issue (no need to retest afterwards): `toolchain` above `sources`/`source_urls`, with empty line in between

we really need a script to check the order of easyconfig parameters according to the style we're trying to uphold...

I won't start on making remarks w.r.t. the funky `version_str` stuff, since I can't think of a better approach (and I know this isn't yours @gjbex).